tests/benchdnn/conv/conv.cpp

```diff
@@ -354,12 +355,20 @@ int fill_src(
 int fill_wei(
         const prb_t *prb, dnn_mem_t &mem_dt, dnn_mem_t &mem_fp, res_t *res) {
     const bool wino_s8 = prb->alg == WINO && prb->cfg[WEI].dt == dnnl_s8;
-    const bool s8_s8
-            = prb->cfg[WEI].dt == dnnl_s8 && prb->cfg[SRC].dt == dnnl_s8;
     const bool is_def_zp = prb->attr.zero_points.is_def(DNNL_ARG_SRC);
     const bool diff_data_type = mem_dt.dt() != mem_fp.dt();
+
+#if DNNL_AARCH64
+    const bool s8_u8
+            = prb->cfg[WEI].dt == dnnl_s8 && prb->cfg[SRC].dt == dnnl_u8;
+    const bool check_reorder
+            = diff_data_type && !wino_s8 && !s8_u8 && is_def_zp;
```

Comment:
Something I would prefer is have a single set of definitions like:
```
#if DNNL_AARCH64
    dnnl_data_type_t dt_check = dnnl_u8;
#else
    dnnl_data_type_t dt_check = dnnl_s8;
#endif
    const bool wei_x8x8
            = prb->cfg[WEI].dt == dnnl_s8 && prb->cfg[SRC].dt == dt_check ;
    const bool check_reorder
            = diff_data_type && !wino_s8 && !wei_x8x8 && is_def_zp;
```

Using `wei_x8x8` instead of conditioning where either `{s8_u8, s8_s8}` are used would reduce the need of adding more `#if/#else` definitions.

Reply:
Thx! I'll change it.